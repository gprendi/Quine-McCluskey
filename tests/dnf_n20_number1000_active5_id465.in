d 20
01000000010000000000
00000001010000000001
00000001000000000000
01000000000000000001
00000000010000000000
00000101000000000001
01000101000000000001
01000001000000000001
01000101000000000000
01000001000000000000
00000100000000000001
01000101010000000000
01000001010000000001
01000001010000000000
00000101010000000000
01000100010000000000
00000100010000000000
00000101010000000001
00000001010000000000
00000000010000000001
00000100000000000000
00000000000000000001
00000100010000000001
01000100000000000000
00000001000000000001
01000100000000000001
01000000010000000001
00000101000000000000
01000000000000000000
01000100010000000001
01000101010000000001
00000000000000000000
