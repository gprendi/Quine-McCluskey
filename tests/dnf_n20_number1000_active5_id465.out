p dnf 20 0
-1 -3 -4 -5 -7 -9 -11 -12 -13 -14 -15 -16 -17 -18 -19 0
